inscriptions/ISic030011.xml

```diff
@@ -109,9 +113,10 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" cert="low" notBefore-custom="-0475" notAfter-custom="-0450">475—450 BCE</origDate> 
+                            <!-- dates from Jordan-->
                         </origin>
-                        <provenance type="found">Found in the archaeological area of Selinunte<geo>37.5853775, 12.8298666</geo>.</provenance> <!-- check provenance -->
+                        <provenance type="found">Found in the archaeological area of Selinunte, in the Malophoros Sanctuary<geo>37.585378, 12.829867</geo>.</provenance> 
```

Comment:
the correct geo ref is: 37.586763, 12.816874 (it is in pleiades: https://pleiades.stoa.org/places/502577471)